index.html

```diff
@@ -3697,149 +3786,181 @@ <h3>The <dfn>ThingFilter</dfn> dictionary</h3>
       </p>
       <pre class="idl">
         dictionary ThingFilter {
-          DiscoveryMethod method = "directory";
-          USVString? url;
           object? fragment;
           <!-- DOMString? query; -->
         };
       </pre>
-      <p class = "ednote">
-        The `DOMString query` property was temporarily removed from
-        <a>ThingFilter</a>, until it is standardized in the WoT Discovery
-        task force.
-      </p>
       <p>
-        The <dfn>method</dfn> property represents the discovery type that should be used in the discovery process. The possible values are defined by the <a href="#the-discoverymethod-enumeration">DiscoveryMethod</a> enumeration.
+        The <dfn>fragment</dfn> property represents a template object used for matching property by property against discovered <a>Thing</a>s.
       </p>
-      <p>
-        The <dfn>url</dfn> property represents the URL of the target entity serving the discovery request, for instance the URL of a <a>Thing Directory</a> (if <a href="#dom-thingfilter-method">method</a> is {{DiscoveryMethod/[["directory"]]}}),
-        or the URL of a directly targeted <a>Thing</a> (if
-        <a href="#dom-thingfilter-method">method</a> is {{DiscoveryMethod/[["direct"]]}})
+      <p class = "ednote">
+        The |query:DOMString| property was temporarily removed from
+        <a>ThingFilter</a>, until it is standardized in the WoT Discovery
+        task force. It represented a query string accepted by the implementation,
+        for instance a SPARQL or JSON query. Support was to be implemented locally
+        in the <a>WoT Runtime</a> or remotely as a service in a <a>Thing Directory</a>.
```

Comment:
IIRC, Thing Directories have been renamed to Thing Description Directories.

```suggestion
        in the <a>WoT Runtime</a> or remotely as a service in a <a>Thing Description Directory</a>.
```